packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -749,4 +749,23 @@ class RenderIndexedStack extends RenderStack {
     super.debugFillProperties(properties);
     properties.add(IntProperty('index', index));
   }
+
+  @override
+  List<DiagnosticsNode> debugDescribeChildren() {
+    return super.debugDescribeChildren().asMap().entries.map<DiagnosticsNode>((MapEntry<int, DiagnosticsNode> node) {
```

Comment:
I think it would be faster and more robust to just generate the list of DiagnosticableTreeNode children for this class rather than seeing to reuse the list from the parent for the visible child.

Reply:
@jacob314 Hey, thanks for the review!

I've pushed a new commit that builds the list.